Make error

Hi, I keep getting the following error when running make in my contrafold-se/src directory.

make: *** [Makefile:49: Contrafold.o] Error 1

I do not have this error when on my mac, but when trying to compile on my Ubuntu desktop I have issues...

Thanks!
Grace

--- a/src/InferenceEngine.hpp
+++ b/src/InferenceEngine.hpp
@@ -21,7 +21,7 @@
 template<class RealT>
 class InferenceEngine
 {
-    static const double DATA_LOW_THRESH = 1e-7;  // used for the data so that log is not -Inf
+    static constexpr double DATA_LOW_THRESH = 1e-7;  // used for the data so that log is not -Inf
 
     const bool allow_noncomplementary;
     unsigned char char_mapping[256];

